community/py-jsonschema/APKBUILD

```diff
@@ -8,7 +8,8 @@ pkgdesc="An implementation of JSON Schema validation for Python"
 url="https://github.com/Julian/jsonschema"
 arch="noarch"
 license="MIT"
-makedepends="python3-dev python2-dev py-setuptools"
+depends=""
+makedepends="python3-dev python2-dev py2-functools32 py-pyrsistent py-attrs py-setuptools"
```

Comment:
Shouldn't the added makedepends be checkdepends?

Reply:
`py-attrs` and `py-jsonschema` have been added already in #7115 (as depends, not makedepends), so they can be skipped here.